### PR TITLE
Walther P99

### PR DESCRIPTION
**Pull request by JoramD0**
**When merged this pull request will:**
- Add the Walther P99 (retexture from vanilla Arma P07).
- Move `vehicleClass` from common macro to individual "vehicles" (due to a different class for pistols).

A few screenshots [here](https://imgur.com/a/CaA51A4) & preview image [here](https://i.imgur.com/KVXsGBL.png).
Let me know if there is anything that can be improved.

### CODE REVIEW

File: addons/weapons/CfgWeapons.hpp [hunk 1]
```diff
@@ -29,6 +29,20 @@ class CfgWeapons {
         baseWeapon = QCLASS(MX_GL_BlackCamo);
     };
 
+    class hgun_P07_F;
+    class CLASS(Walther_P99): hgun_P07_F
+    {
```
Comment (jonpas) [inline addons/weapons/CfgWeapons.hpp]: On same line as class definition.

File: addons/weapons/CfgWeapons.hpp [hunk 2]
```diff
@@ -29,6 +29,20 @@ class CfgWeapons {
...
+        dlc = QUOTE(PREFIX);
+        scope = 2;
+        author = "JoramD";
+        displayName = CSTRING(Walther_P99);
+        picture = QPATHTOF(UI\walther_p99_ca.paa);
+        hiddenSelections[] = {"camo"};
+        hiddenSelectionsTextures[] = {
+            QPATHTOF(data\walther_p99_co.paa)
```
Comment (jonpas) [inline addons/weapons/CfgWeapons.hpp]: Can keep this on the same line, since it's only one.

File: addons/weapons/stringtable.xml [hunk 3]
```diff
@@ -16,5 +16,10 @@
             <German>MX 3GL 6,5 mm (schwarz tarn)</German>
             <Japanese>MX 3GL 6,5 mm (黒)</Japanese>
         </Key>
+        <Key ID="STR_TACS_Weapons_Walther_P99">
+            <English>Walther P99</English>
+            <German></German>
```
Comment (jonpas) [inline addons/weapons/stringtable.xml]: Just delete the unused ones.

**Comment by JoramD0** [comment]
Attached file has a psd file with the preview images may someone need them.

[Walther_P99_Preview.zip](https://github.com/Theseus-Aegis/TheseusServices/files/2066172/Walther_P99_Preview.zip)

**Pull request merged**